frb_example/pure_dart/rust/Cargo.toml

```diff
@@ -8,7 +8,12 @@ crate-type = ["cdylib"]
 
 [dependencies]
 anyhow = { version = "1.0.64", features = ["backtrace"] }
-flutter_rust_bridge = { path = "../../../frb_rust", features = ["chrono", "uuid", "backtrace"] }
+flutter_rust_bridge = { path = "../../../frb_rust", features = [
+    "backtrace",
+    "chrono",
+    "dart-opaque",
```

Comment:
nit: given that dart_opaque is a default feature, maybe we can keep these lines unchanged